lib/fizzy/parser_expr.cpp

```diff
@@ -92,9 +106,84 @@ parser_result<Code> parse_expr(const uint8_t* pos)
         case Instr::i64_extend_i32_u:
             break;
 
+        case Instr::end:
+        {
+            if (!open_blocks_possitions.empty())
```

Comment:
Need to also add an assertion at the end of `parse_expr` for `positions.empty()` (so that all of them were processed, e.g. it is balanced).